orebfuscator-plugin/src/main/java/net/imprex/orebfuscator/proximity/ProximityDirectorThread.java

```diff
@@ -12,6 +12,7 @@
 import org.bukkit.Bukkit;
 import org.bukkit.entity.Player;
 import org.bukkit.event.EventHandler;
+import org.bukkit.event.EventPriority;
```

Comment:
No you didn't specify an EventPriority anywhere in this class which makes this an unused import. Their also shouldn't be an EventPriority on the PlayerJoinEvent since their is no need to change anything here.

Reply:
oh, didn't notice this file